src/ClientHandle.cpp

```diff
@@ -1424,6 +1424,11 @@ void cClientHandle::HandleRightClick(int a_BlockX, int a_BlockY, int a_BlockZ, e
 	cPluginManager * PlgMgr = cRoot::Get()->GetPluginManager();
 
 	bool Success = false;
```

Comment:
I guess this variable can be removed entirely now right?

Reply:
Actually true. I somehow thought that this variable is used as a return value.